docs/usage/advanced.rst

```diff
@@ -95,3 +95,42 @@ it is enough to set log level to the desired value.
     logging.basicConfig()
     root_logger = logging.getLogger()
     root_logger.setLevel(logging.DEBUG)
+
+Observing HTTP traffic
+----------------------
+
+There are cases where you need access to the transport protocol (http). For
+example: you need to read value of specific http header. Pyodata provides
+simple mechanism to observe all http requests and access low lever properties
+directly from underlying engine (**python requests**).
```

Comment:
This is not true. Pyodata does not have underlying engine (**python requests**) - it is networking library agnostic, dependency to requests is not direct in requirements.txt, by readme https://github.com/SAP/python-pyodata#usage it only expect Requests library Session-like object. Pyodata can work with other http networking libraries.

Reply:
The only issue with the paragraph is that it mentions "python requests", otherwise I see no untrue parts :)

Reply:
that was the only line this comment was referring to :)

Reply:
I slightly rephrased this last sentence. Reference to python requests is sill there, but just as an example.